x-pack/plugins/security_solution/public/timelines/components/timeline/body/helpers.test.ts

```diff
@@ -246,56 +245,6 @@ describe('helpers', () => {
     });
   });
 
-  describe('isInvestigateInResolverActionEnabled', () => {
```

Comment:
moved to `.../detections/components/alerts_table/timeline_actions/investigate_in_resolver.test.tsx`